gcp/disk.go

```diff
@@ -9,7 +9,7 @@ import (
 	compute "google.golang.org/api/compute/v1"
 )
 
-const GBbytes = 1_000_000_000
+const GiBbytes = 1073741824
```

Comment:
Just for the sake of readibility let's use thousands separators:
```suggestion
const GiBbytes = 1_073_741_824
```

Reply:
TIL!, such a grOIT feature